README.md

```diff
@@ -68,6 +68,13 @@ You should probably use this README.dist.md as a base for your project's README.
 mv README.{dist.md,md}
 ```
 
+Also, in order to improve your usage of invoke scripts, you can install console autocompletion.
+```bash
+invoke --print-completion-script=MY_SHELL > ~/.bin/invoke-completion.bash
+```
+
+Replace `MY_SHELL` with the shell you're using (it's supporting `bash`, `zsh` & `fish` shells) and source the script in your `.bashrc` file.
```

Comment:
That's awesome, and should even be in the real README.

Maybe listed as an optional requirement here: https://github.com/jolicode/docker-starter/blob/master/README.dist.md#requirements

Reply:
I agree this could goes to the .dist